src/coreclr/jit/emitfmtsarm64_sve.h

```diff
@@ -0,0 +1,514 @@
+IF_DEF(SVE_DV_4A,   IS_NONE, NONE) // SVE_DV_4A  ........ix.xxxvv ..NNNN.MMMM.DDDD  -- sve_int_pred_dup
+IF_DEF(SVE_DW_2A,   IS_NONE, NONE) // SVE_DW_2A  ........xx...... ......iiNNN.DDDD  -- sve_int_ctr_to_mask
+IF_DEF(SVE_DW_2B,   IS_NONE, NONE) // SVE_DW_2B  ........xx...... .......iNNN.DDDD  -- sve_int_ctr_to_mask
+IF_DEF(SVE_DX_3A,   IS_NONE, NONE) // SVE_DX_3A  ........xx.mmmmm ......nnnnn.DDD.  -- sve_int_while_rr_pair
+IF_DEF(SVE_DY_3A,   IS_NONE, NONE) // SVE_DY_3A  ........xx.mmmmm ..l...nnnnn..DDD  -- sve_int_while_rr_pn
+IF_DEF(SVE_DZ_1A,   IS_NONE, NONE) // SVE_DZ_1A  ........xx...... .............DDD  -- sve_int_pn_ptrue
+IF_DEF(SVE_EA_1A,   IS_NONE, NONE) // SVE_EA_1A  ........xx...... ...iiiiiiiiddddd  -- sve_int_dup_fpimm
+IF_DEF(SVE_EB_1A,   IS_NONE, NONE) // SVE_EB_1A  ........xx...... ..hiiiiiiiiddddd  -- sve_int_dup_imm
+IF_DEF(SVE_EB_1B,   IS_NONE, NONE) // SVE_EB_1B  ........xx...... ...........ddddd  -- sve_int_dup_imm
+IF_DEF(SVE_EC_1A,   IS_NONE, NONE) // SVE_EC_1A  ........xx...... ..hiiiiiiiiddddd  -- sve_int_arith_imm0
+IF_DEF(SVE_ED_1A,   IS_NONE, NONE) // SVE_ED_1A  ........xx...... ...iiiiiiiiddddd  -- sve_int_arith_imm1
+IF_DEF(SVE_EE_1A,   IS_NONE, NONE) // SVE_EE_1A  ........xx...... ...iiiiiiiiddddd  -- sve_int_arith_imm2
+IF_DEF(SVE_EF_3A,   IS_NONE, NONE) // SVE_EF_3A  ...........mmmmm ......nnnnnddddd  -- sve_intx_dot2
+IF_DEF(SVE_EG_3A,   IS_NONE, NONE) // SVE_EG_3A  ...........iimmm ......nnnnnddddd  -- sve_intx_dot2_by_indexed_elem
+IF_DEF(SVE_EH_3A,   IS_NONE, NONE) // SVE_EH_3A  ........xx.mmmmm ......nnnnnddddd  -- sve_intx_dot
+IF_DEF(SVE_EI_3A,   IS_NONE, NONE) // SVE_EI_3A  ...........mmmmm ......nnnnnddddd  -- sve_intx_mixed_dot
+IF_DEF(SVE_EJ_3A,   IS_NONE, NONE) // SVE_EJ_3A  ........xx.mmmmm ....rrnnnnnddddd  -- sve_intx_cdot
+IF_DEF(SVE_EK_3A,   IS_NONE, NONE) // SVE_EK_3A  ........xx.mmmmm ....rrnnnnnddddd  -- sve_intx_cmla
+IF_DEF(SVE_EL_3A,   IS_NONE, NONE) // SVE_EL_3A  ........xx.mmmmm ......nnnnnddddd  -- sve_intx_mlal_long
+IF_DEF(SVE_EM_3A,   IS_NONE, NONE) // SVE_EM_3A  ........xx.mmmmm ......nnnnnddddd  -- sve_intx_qrdmlah
+IF_DEF(SVE_EN_3A,   IS_NONE, NONE) // SVE_EN_3A  ........xx.mmmmm ......nnnnnddddd  -- sve_intx_qdmlalbt
+IF_DEF(SVE_EO_3A,   IS_NONE, NONE) // SVE_EO_3A  ........xx.mmmmm ......nnnnnddddd  -- sve_intx_qdmlal_long
+IF_DEF(SVE_EP_3A,   IS_NONE, NONE) // SVE_EP_3A  ........xx...... ...gggmmmmmddddd  -- sve_intx_pred_arith_binary
+IF_DEF(SVE_EQ_3A,   IS_NONE, NONE) // SVE_EQ_3A  ........xx...... ...gggnnnnnddddd  -- sve_intx_accumulate_long_pairs
+IF_DEF(SVE_ER_3A,   IS_NONE, NONE) // SVE_ER_3A  ........xx...... ...gggmmmmmddddd  -- sve_intx_arith_binary_pairs
+IF_DEF(SVE_ES_3A,   IS_NONE, NONE) // SVE_ES_3A  ........xx...... ...gggnnnnnddddd  -- sve_intx_pred_arith_unary
+IF_DEF(SVE_ET_3A,   IS_NONE, NONE) // SVE_ET_3A  ........xx...... ...gggmmmmmddddd  -- sve_intx_pred_arith_binary_sat
```

Comment:
SVE_ET_3A and SVE_EU_3A look identical?

Reply:
Yes, they could be and so are other. They are given unique names based on "format pattern + SVE group class", so there might be format names that represent same format, but different group (SVE_ET_3A = `SVE2 saturating add/subtract` vs. SVE_EU_3A  = `SVE2 saturating/rounding bitwise shift left (predicated)`). They can (will) be hand combined once we start implementing the encoding logic.